app/controllers/core_induction_programme/lessons_controller.rb

```diff
@@ -22,20 +22,25 @@ def show
   def edit; end
 
   def update
-    @course_lesson.save!
-    flash[:success] = "Your changes have been saved"
-    redirect_to cip_year_module_lesson_url
+    @course_lesson.assign_attributes(course_lesson_params)
+
+    if @course_lesson.valid?
```

Comment:
Just doing this lesson renders ActiveRecord::RecordInvalid in CoreInductionProgramme::LessonsController#update page rather than edit again with the error. Should we do something to catch this error?

Reply:
Hmm, `update(...)` or `update!(...)`? The latter I expect to throw, the former I expect to return a boolean

Reply:
If they don't work just keep the existing syntax.

Reply:
Thought i'd tried both variations initially but double checked and yeah, it did as you said. i'll put it to .update(course_lesson_params)